cfgov/jinja2/v1/_layouts/base-common.html

```diff
@@ -23,10 +23,10 @@
 -->
 {% endif %}
 
-<!--[if IE 7]>         <html lang="en" class="no-js lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html lang="en" class="no-js lt-ie10 lt-ie9"> <![endif]-->
-<!--[if IE 9]>         <html lang="en" class="no-js lt-ie10"> <![endif]-->
-<!--[if gt IE 8]><!--> <html lang="en" class="no-js"> <!--<![endif]-->
+<!--[if IE 7]>         <html lang="{{  language  }}" class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html lang="{{  language  }}" class="no-js lt-ie10 lt-ie9"> <![endif]-->
+<!--[if IE 9]>         <html lang="{{  language  }}" class="no-js lt-ie10"> <![endif]-->
+<!--[if gt IE 8]><!--> <html lang="{{  language  }}" class="no-js"> <!--<![endif]-->
```

Comment:
Gonna be super-nitpicky, but we usually only have a single space after `{{` and before `}}`.

Reply:
Fix'd